experimental/packages/otlp-transformer/package.json

```diff
@@ -42,11 +42,11 @@
     "README.md"
   ],
   "peerDependencies": {
-    "@opentelemetry/api": ">=1.1.0 <1.2.0",
+    "@opentelemetry/api": ">=1.0.0 <1.2.0",
     "@opentelemetry/api-metrics": "~0.27.0"
```

Comment:
Is it intended to have api-metrics as peer dependency here but as dependency in opentelemetry-sdk-metrics-base?